lib/ProductOpener/Products.pm

```diff
@@ -2414,6 +2419,23 @@ sub index_product($product_ref)
 	return;
 }
 
+sub index_search_service($product_ref) {
+	if ($ProductOpener::Config2::redis_url eq "")
+	{
+		$log->warn("Redis URL not provided for search indexing", { error => $@ }) if $log->is_warn();
+		return;
+	}
+	# Now send the barcode to Redis so that it can be indexed
+	eval {
+		my $redis_client = Redis::Client->new(host => $ProductOpener::Config2::redis_url);
```

Comment:
Shouldn't we reuse the redis connection ?

I would rather (even if it's short) put that in a specific file: Redis.pm and have a global for $redis_client.
Then use get_redis_client() to initialize it, if it's not already initialized.

This sub could also be in Redis.pm, for the moment.